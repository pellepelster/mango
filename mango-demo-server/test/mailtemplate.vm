Hello ${name}, 

welcome to ${link}, your username is ${username}