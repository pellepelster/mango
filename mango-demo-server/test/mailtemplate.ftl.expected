Hello Steve,

welcome to http://www.example.org, your username is steve23
